static/app/views/replays/detail/console/index.tsx

```diff
@@ -59,17 +66,26 @@ function Console({breadcrumbs, startTimestamp = 0}: Props) {
 
       {filteredBreadcrumbs.length > 0 ? (
         <ConsoleTable>
-          {filteredBreadcrumbs.map((breadcrumb, i) => (
-            <ConsoleMessage
-              relativeTimestamp={showPlayerTime(
-                breadcrumb.timestamp || '',
-                startTimestamp
-              )}
-              key={i}
-              isLast={i === breadcrumbs.length - 1}
-              breadcrumb={breadcrumb}
-            />
-          ))}
+          {filteredBreadcrumbs.map((breadcrumb, i) => {
+            const diff = relativeTimeInMs(breadcrumb.timestamp || '', startTimestamp);
+            const onClick = () => setCurrentTime(diff);
+            const onMouseOver = () => setCurrentHoverTime(diff);
+            const onMouseOut = () => setCurrentHoverTime(undefined);
+            return (
+              <ConsoleMessage
+                onClick={onClick}
+                onMouseOver={onMouseOver}
+                onMouseOut={onMouseOut}
+                relativeTimestamp={showPlayerTime(
+                  breadcrumb.timestamp || '',
+                  startTimestamp
+                )}
```

Comment:
We should probably have `startTimestamp` as a prop for `ConsoleMessage` and calculate relativeTimestamp inside of that component.